aircon/aircon.py

```diff
@@ -174,11 +183,6 @@ def queue_command(self, name: str, value) -> None:
       data_value = data_value.value
       typed_value = data_type[value]
 
-    # Update value precision for value to be sent to the A/C
-    precision = self._properties.get_precision(name)
```

Comment:
Since precision only makes sense if the property is an `int` anyway, I've moved this to the int clause above